### PR TITLE
Start adding sponsorship info to README

### PR DESCRIPTION
**Pull request by jenncoop**
Adding the Sponsorship Package, finally!

### CODE REVIEW

**Comment by whilestevego** [comment]
It looks good. I'm curious how you chose the numbers? Do you think we should also have tiers for recurring sponsorships in the future? :no_mouth: :thought_balloon:

**Comment by expede** [comment]
Any thoughts on splitting this into its own file? Much like how many repos split out `LICENSE` and `CONTRIBUTING` docs

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
-# sponsorship-package
-Code and Coffee Vancouver's Sponsorship Package
+# Sponsorship Package
+
+##Tiers:
+1) Venue Sponsor: FREE
```
Comment (expede) [inline README.md:4]: Perhaps `Venue Sponsor: Provide a venue`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
-# sponsorship-package
-Code and Coffee Vancouver's Sponsorship Package
+# Sponsorship Package
+
+##Tiers:
```
Comment (expede) [inline README.md:3]: `Monthly Tiers`, maybe?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+3) Child Care Sponsor: $200
+- Logo on the meetup page leading up to the event
+- Company write up in the event invite
+- Can put out company garb at event
+- Thanked during intro by organizer
+
+4) Event Sponsor: $500
+- Logo on the meetup page leading up to the event
+- Company write up in the event invite
+- Can put out company garb at event
```
Comment (expede) [inline README.md]: "company garb"? Like, swag?

**Comment by expede** [comment]
I agree with @whilestevego on recurring sponsorship tiers

**Comment by expede** [comment]
We probably will want a document on what sponsor we will accept (i.e.: don't want to take money from horrible people who do horrible things)

**Comment by expede** [comment]
@whilestevego #2

File: README.md (hunk 3 above)
Comment (jenncoop) [inline README.md]: Swag is a much better word

**Comment by jenncoop** [comment]
Thanks for the comments. Will make adjustments and pass it back.

**Comment by expede** [comment]
Cool. Thanks so much for writing this up @jenncoop!

**Comment by whilestevego** [comment]
Indeed. Thank you so much, @jenncoop!

**Comment by abinyah** [comment]
I agree with all the above. But everything should be $1 less, just because it looks better.
In regards to Brooks' comment, we should also just have a general guideline on who we should solicit to be sponsors and for what tier.

**Pull request merged**